discord_bot/discord_bot.py

```diff
@@ -144,24 +142,45 @@ def load_config(config_file_path):
                     self.logger.warning("Invalid Session")
                 elif message["op"] == Opcodes.DISPATCH:
                     event = message["t"]
-                    if event == Events.READY:
+                    if event == Events.READY.value:
                         self.session_id = message["d"]["session_id"]
+                    elif event == Events.MESSAGE_CREATE.value:
+                        channel_id = message["d"]["channel_id"]
+                        author = message["d"]["author"]["username"]
+                        content = message["d"]["content"]
+                        await self.respond_message(channel_id, author, content)
                 else:
                     self.logger.exception("Unexpected opcode {}: {}".format(message["op"], message))
 
     # TODO: Remove this test code
-    # async def send_message(self, recipient_id, content):
-    #     """
-    #     Post a message into the given channel
-    #     :param recipient_id: num - the recipient to open a DM channel with
-    #     :param content: obj - message sent
-    #     :return: dict - Fires a "Message Create" Gateway event
-    #     """
-    #     channel = requests.post("{}", json={"recipient_id": recipient_id}).json()
-    #
-    #     return requests.post(
-    #         "{}/channels/{}/messages".format(
-    #             self.config["discord_api_endpoint"], channel["id"]),
-    #         json={
-    #             "content": content
-    #         }).json()
+    async def respond_message(self, channel_id, author, content):
+        """
+        Receives a message and responds to the user in the same channel.
+        :param channel_id: int - channel id
+        :param author: string - author of the message
+        :param content: string - message the user inputs
+        :return:
+        """
+        defaults = {
+            "headers": {
+                "Authorization": "Bot {}".format(self.config["handshake_identity"]["token"]),
+                "User-Agent": "MomoiBot (https://github.com/ShifuYee/Discord-Bot)"
+            }
+        }
+        kwargs = dict(defaults, data={
+            "content": f"{author} how may I help?"
+        })
+        if content.startswith("!help"):
+            url = "{}/channels/{}/messages".format(self.config["discord_api_endpoint"], channel_id)
+            async with ClientSession() as session:
+                async with session.request(
```

Comment:
If it easily fits on one line, you should probably leave it as one line, or it looks more complicated than it needs to be.